library/src/main/java/com/chuckerteam/chucker/internal/support/DateJsonAdapter.kt

```diff
@@ -11,8 +11,8 @@ class DateJsonAdapter {
     private val dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, Locale.US)
 
     @ToJson
-    fun toJson(date: Date): String = dateFormat.format(date)
+    fun toJson(date: Date?): String? = date?.let { dateFormat.format(it) }
```

Comment:
Can you explain why you decided to do nullable parameter and return value here nullable?

Reply:
Removed code related to `Date` parsing as it is not used anywhere in the codebase.